prow/config.yaml

```diff
@@ -166,7 +166,7 @@ presubmits:
           path: /mnt/disks/ssd0
   - name: pull-federation-e2e-gce
     agent: jenkins
-    always_run: false
+    always_run: true
     context: pull-federation-e2e-gce
     rerun_command: "/test pull-federation-e2e-gce"
     trigger: "(?m)^/test pull-federation-e2e-gce,?(\\s+|$)"
```

Comment:
this needs to be `"(?m)^/test( all| pull-federation-e2e-gce),?(\\s+|$)"` now that it is `always_run: true`

Reply:
fixed.